ibazel/ibazel.go

```diff
@@ -324,6 +325,7 @@ func (i *IBazel) iteration(command string, commandToRun runnableCommand, targets
 		log.Logf("%s %s", strings.Title(verb(command)), joinedTargets)
 		i.beforeCommand(targets, command)
 		outputBuffer, err := commandToRun(targets...)
+		i.interruptCount = 0
```

Comment:
Heh, this is subtle. I bet you pulled your hair out trying to figure out why it killed after just one `ctrl-c` when you hit this the first time 😉 sorry!

Reply:
Luckily I'm bald 😄